src/Weapsy/appsettings.json

```diff
@@ -3,7 +3,7 @@
     "InstrumentationKey": ""
   },
   "ConnectionStrings": {
-    "DefaultConnection": "Data Source=.\\SQLExpress;Initial Catalog=weapsy.dev;Integrated Security=SSPI"
+    "DefaultConnection": "Data Source=localhost;Initial Catalog=weapsy.dev;user id=sa;password=111;Integrated Security=SSPI"
```

Comment:
There is no need to change the connection string at the moment.
Please discard this change and I will approve the request.